integration/report_nodes_test.go

```diff
@@ -27,7 +27,7 @@ func TestReportCommand_Nodes(t *testing.T) {
 	out, err, exitcode := ChefAnalyzeWithCredentials("report", "nodes")
 	assert.Contains(t,
 		out.String(),
-		"Node Name                 Chef Version    Operating System    Cookbooks",
+		"No nodes found to analyze.",
```

Comment:
To ensure we continue to cover the case of when data is present, please add a test that verifies headers when there is at least one node available.